scripts/fddaqconf_gen

```diff
@@ -436,7 +436,8 @@ def cli(
         daq_common=daq_common,
         tp_infos=tp_infos,
         trigger_data_request_timeout=trigger_data_request_timeout,
-        use_hsi_input=(hsi.use_fake_hsi or hsi.use_timing_hsi or ctb_hsi.use_ctb_hsi),
+        use_hsi_input=(hsi.use_fake_hsi or hsi.use_timing_hsi),
+        use_ctb_input=ctb_hsi.use_ctb_hsi,
```

Comment:
Looks fine, though I imagine this will change to have a separate `use_timing_hsi`, or even `use_dts_hsi` in the future then the changes in hsi-gen code by @strilov happen.